### PR TITLE
fixed high res issue on sell page

### PR DESCRIPTION
**Pull request by EagleEye25**

### CODE REVIEW

**Comment by kewde** [comment]
I think this code will clear the page on every resize and load the first page.

File: src/app/market/sell/sell.component.ts [hunk 1]
```diff
@@ -68,13 +68,17 @@ export class SellComponent implements OnInit {
 
   public search: string = '';
   public category: string = '';
+  screenHeight: number;
+  screenWidth: number;
```
Comment (kewde) [inline src/app/market/sell/sell.component.ts]: rm

File: src/app/market/sell/sell.component.ts [hunk 2]
```diff
@@ -110,7 +114,8 @@ export class SellComponent implements OnInit {
     this.isLoading = true;
     const search = this.filters.search ? this.filters.search : '*';
     const hashItems = this.filters.hashItems ? this.filters.hashItems === 'true' : undefined;
-    const max = this.pagination.maxPerPage;
+    let max: number;
+    this.screenHeight > 1330 ? max = 20 : max = this.pagination.maxPerPage;
```
Comment (kewde) [inline src/app/market/sell/sell.component.ts]: rm, just set `this.pagination.maxPerPage` equal to 20 when resized

File: src/app/market/sell/sell.component.ts [hunk 3]
```diff
@@ -209,4 +214,12 @@ export class SellComponent implements OnInit {
     this.pages[pageIndex].listings.splice(listingIndex, 1);
   }
 
+  @HostListener('window:resize', ['$event'])
+  getScreenSize() {
+    this.screenHeight = window.innerHeight;
+    this.screenWidth = window.innerWidth;
```
Comment (kewde) [inline src/app/market/sell/sell.component.ts]: rm

File: src/app/market/sell/sell.component.ts [hunk 4]
```diff
@@ -209,4 +214,12 @@ export class SellComponent implements OnInit {
...
+    if (this.screenHeight > 1330) {
```
Comment (kewde) [inline src/app/market/sell/sell.component.ts]: Maybe something like this?
```
this.pagination.maxPerPage = window.innerWidth > 1330 ? 20 : 10;

if (screenIsNowLarger) {
   // load next page
   this.loadNextPage();
} 
```

**Review by kewde — CHANGES_REQUESTED** [review]

File: src/app/market/sell/sell.component.ts [hunk 5]
```diff
@@ -209,4 +214,12 @@ export class SellComponent implements OnInit {
...
+      this.loadPage(0, true);
```
Comment (kewde) [inline src/app/market/sell/sell.component.ts]: This will reset everything, so if you do it midway a search, you lose stuff

**Review by zaSmilingIdiot — APPROVED** [review]

**Pull request merged**